invokeai/app/services/config/config_default.py

```diff
@@ -28,11 +28,11 @@
 DEFAULT_VRAM_CACHE = 0.25
 DEVICE = Literal["auto", "cpu", "cuda", "cuda:1", "mps"]
 PRECISION = Literal["auto", "float16", "bfloat16", "float32"]
-ATTENTION_TYPE = Literal["auto", "normal", "xformers", "sliced", "torch-sdp"]
+ATTENTION_TYPE = Literal["auto", "sliced", "torch-sdp"]
 ATTENTION_SLICE_SIZE = Literal["auto", "balanced", "max", 1, 2, 3, 4, 5, 6, 7, 8]
```

Comment:
I agree with @psychedelicious that since "balanced" and "auto" are equivalent, we should deprecate one. I suggest deprecating balanced and keeping auto to be consistent with "auto" configs elsewhere.

Reply:
I'll ask @psychedelicious too - should we remove `auto`?
Because I feel that `balanced` will describe logic better, while `auto` initially was for 'Invoke will decide what to do itself', we currently not doing this way, we simply run sliced on half of available heads in attention, balanced compromise between speed and memory consumption.